docs/form_builder.md

```diff
@@ -181,6 +181,10 @@ rake form_definition:validate_all
 
 This will validate all forms in directories `["config/forms", "spec/fixtures/forms"]`
 
+## Form models and definition
+
+For information about the form model and related models (section, subsection, page, question) and how these relate to each other follow [this link](docs/form/form.md)
```

Comment:
Actually this link doesn't work because you're using relative path. Think it needs to be either `form/form.md` or /docs/form/form.md`)